### DISCUSSION TITLE
Google Street View crawling without using the Google Street View API

### DISCUSSION THREAD

**Discussion by Tomckk** [community/community#77291] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Is there a way to get Google Street View images without using Google Street View API? 
 Is it possible to get Street View data from some publicly available datasets?
 I really hope to get an answer.

**Comment by Akash1134** [comment]
Hi @Tomckk ,

The GitHub Community on Discussions is a space for all GitHub users to come and discuss ideas and projects. Your post was removed because it contained content that violates our Community Code of Conduct, including posting the same comment in multiple places. As a result, we have hidden the duplicate comments, and we ask that you no longer cross-post.

For additional guidance, please review our [Community on Discussions Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md) and [GitHub’s Terms of Service](https://docs.github.com/en/github/site-policy/github-terms-of-service).
